src/qtiCommonRenderer/renderers/interactions/GraphicGapMatchInteraction.js

```diff
@@ -33,39 +33,38 @@ import instructionMgr from 'taoQtiItem/qtiCommonRenderer/helpers/instructions/in
 import interact from 'interact';
 import interactUtils from 'ui/interactUtils';
 
-var isDragAndDropEnabled;
+let isDragAndDropEnabled;
 
 // this represents the state for the active droppable zone
 // we need it only to access the active dropzone in the iFrameFix
 // should be removed when the old test runner is discarded
-var activeDrop = null;
+let activeDrop = null;
 
 /**
- * Global variable to count number of choice usages:
+ * Global constiable to count number of choice usages:
```

Comment:
**chore**: Typo! Maybe a too-greedy search and replace :). I used to search for `var ` instead of `var` to avoid this problem...